docs/modules/ROOT/pages/api/erc721.adoc

```diff
@@ -731,6 +689,147 @@ See <<IERC721-ApprovalForAll,IERC721::ApprovalForAll>>.
 
 See <<IERC721-Transfer,IERC721::Transfer>>.
 
+== Receiver
```

Comment:
I'm just curious here. Any reason why you decided to put the Receiver section here before the Extensions?

Reply:
Extension felt more generic, and I wanted to prioritize the receiver modules since they are part of the core flow.

Reply:
Should we include them both in core? They're both defined in the EIP and that's how it's set up in the [solidity docs](https://docs.openzeppelin.com/contracts/5.x/api/token/erc721)